dataprofiler/profilers/profiler_options.py

```diff
@@ -761,3 +761,61 @@ def _validate_helper(self, variable_path='ProfilerOptions'):
             variable_path=variable_path + '.structured_options')
 
         return errors
+
+
+class TextProfilerOptions(BaseInspectorOptions):
+
+    def __init__(self, is_enabled=True):
```

Comment:
should also include: `is_case_sensitive`, `stop_words`, not sure about dealing with `words` and `voacb`

Reply:
not sure it's needed now. At least some other classes don't have the default values now. And like you said, I don't know how to make it for words and vocab.

Reply:
`is_case_sensitive` and `stop_words` should still be able to be set through the constructor